src/components/Home/Home.jsx

```diff
@@ -0,0 +1,35 @@
+import "./Home.css"
+import { useEffect, useState } from "react"
+import BookListing from "../Booklisting/Booklisting"
+
+function Home() {
+    const [books, setBooks] = useState ([])
+    const [clickedBookId, setclickedBookId] = useState(null)

```

Comment:
It doesn't look like this is being used at all, if it isn't it should br removed.